content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -106,13 +106,10 @@ is busy handling the request, the worker won't start handling the next request
 immediately. This process adds to the queue wait time of the next request and affects
 average and tail latencies. The actual tail latency depends on the traffic pattern.
 
-Inside the mesh, a request traverses the client-side proxy and then the server-side
-proxy. This two proxies on the data path add about 6.3 ms to the 90th percentile latency at 1000 requests per second.
-The server-side proxy alone adds 1.7 ms to the 90th percentile latency.
-
 ### Latency for Istio {{< istio_release_name >}}
 
-The default configuration of Istio {{< istio_release_name >}} adds 6.3 ms to the 90th percentile latency of the data plane over the baseline.
+Inside the mesh, a request traverses the client-side proxy and then the server-side
```

Comment:
Default == telemetry v2 , right?

Reply:
I treat it as both mode, but for 1.5 our default is telemetry v2, then the number is `2.8 ms and 2.7 ms` for 90th and 99th separately. Should I add a small explanation like" The default (i.e. telemetryv2)`?

Reply:
Yes please do add explanation, Since we are changing default we should specify it here.
Default should now be v2 telemetry.